### PR TITLE
WH plotting

### PR DESCRIPTION
**Pull request by lucalavezzo**
Some things I still want to fix:

- [x] submit.py slurm option should use the suepRun instead of the user-specific environment
- [x] need to make sure changes to make_plots.py etc. don't break other analyses

### CODE REVIEW

File: plotting/submit.py
Comment (chadfreer) [inline plotting/submit.py:1]: We should add default in the argparse to make this run multithread.

File: setup.sh
Comment (chadfreer) [inline setup.sh:1]: We should not bind /cvmfs to apptainer

File: plotting/plot_utils.py
Comment (chadfreer) [inline plotting/plot_utils.py:1]: Might be worth combining the TTbar and the SingleTops here

File: kraken_run.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def main():
         file_ext = "hdf5"
     elif options.WH == 1:
         condor_file = "condor_SUEP_WH.py"
-        outfile = "out"
+        outfile = "condor_out"
```
Comment (chadfreer) [inline kraken_run.py:164]: Why this distinction only for the WH? Keep this the same unless necessary

File: condor_SUEP_WH.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
     runner = processor.Runner(
         executor=processor.FuturesExecutor(compression=None, workers=1),
         schema=processor.NanoAODSchema,
-        xrootdtimeout=60,
+        xrootdtimeout=120,
```
Comment (chadfreer) [inline condor_SUEP_WH.py:47]: Why is this huge timeout necessary for WH?

Comment (lucalavezzo) [reply condor_SUEP_WH.py:47]: This is more about our T2's xrootd than WH or ggF.

File: kraken_run.py (hunk 1 above)
Comment (lucalavezzo) [reply kraken_run.py:164]: Ah sorry, this is because the WH code uses the pandas_accumulator which calls things like this. Let me standardize this.

**Pull request merged**